### PR TITLE
test(versions): bump salt pkg versions for travis

### PR DESCRIPTION
**Pull request by noelmcloughlin**
### PR progress checklist (to be filled in by reviewers)

- [x] Changes to documentation are appropriate (or tick if not required)
- [x] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [x] `[ci]`       Changes to the continuous integration configuration
- [ ] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [ ] `[docs]`     Documentation changes
- [x] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Describe the changes you're proposing

Bump salt package versions for Travis

### Documentation checklist
- [x] Updated the `README` (e.g. `Available states`).
- [x] Updated `pillar.example`.

### Testing checklist

- [x] Included in Kitchen (i.e. under `state_top`).
- [x] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [x] Updated the relevant test pillar.

### CODE REVIEW

**Comment by myii** [comment]
Thanks for this @noelmcloughlin.  I'm just running a test across all of the instances to ensure the changes are working properly:

* https://travis-ci.org/myii/salt-formula/builds/641153872

**Comment by noelmcloughlin** [comment]
Thanks @myii for doing that.  I reviewed and added a commit to fix Centos.  I must look at Arch but I think thats unrelated issue.

**Comment by myii** [comment]
@noelmcloughlin Don't worry about Arch, it's not supported by this formula.  I've got a couple of suggestions that I'll be adding as a review shortly.

**Comment by noelmcloughlin** [comment]
> it's not supported by this formula

Arch is supported (well it can work). I've been using this on Arch successfully in  2019

**Comment by noelmcloughlin** [comment]
> 've got a couple of suggestions that I'll be adding as a review shortly.

Not too many. I've been working on this formula all day (to clear the way)  so done my bit ;-)

File: test/integration/v201902-py2/controls/pkgs_spec.rb [hunk 1]
```diff
@@ -3,9 +3,18 @@
...
+    case platform[:name]
+    when 'amazon'
+      '2019.2.3-1.el7'
+    when 'centos'
+      '2019.2.3-1.el7'
+    end
+  when 'fedora'
+    '2019.2.0-1.fc30'
+  when 'suse'
+    '2019.2.0-lp151.5.9.1'
```
Comment (myii) [inline test/integration/v201902-py2/controls/pkgs_spec.rb]: Only the one original line needs to be changed:

```diff
+    '2019.2.3-1.el7'
-    case platform[:name]
-    when 'amazon'
-      '2019.2.3-1.el7'
-    when 'centos'
-      '2019.2.3-1.el7'
-    end
-  when 'fedora'
-    '2019.2.0-1.fc30'
-  when 'suse'
-    '2019.2.0-lp151.5.9.1'
```

Our Fedora and OpenSuse pre-salted images are Python 3 only, so don't need to be tested here (`py2`).

**Comment by myii** [comment]
@noelmcloughlin Do you mind squashing the commits into the single original commit, for `semantic-release` purposes, i.e.

* `test(versions): bump salt pkg versions`

Otherwise, I can `Squash and merge` using GitHub instead.

**Comment by myii** [comment]
> `test/integration/v201902-py3/controls/pkgs_spec.rb`:
> 
> ```diff
>      when 'centos'
> -      '2019.2.2-1.el8'
> +      '2019.2.3-1.el8'
> ```

@noelmcloughlin I've added the above in another commit and I'll merge this once Travis is done.

**Pull request merged**